src/App.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react'
+import SplitPane from 'react-split-pane'
+
+import Overlay from './components/preview'
+import EditorPanel from './components/settingsMenu'
+// import onClike from './components/generate'
+
+
+import { SettingsContext } from './lib/contexts'
+import { loadCss, loadStorage } from './lib/utils'
+
+import './App.css'
+
+import {
+  GURMUKHI,
+  TRANSLITERATION_ENGLISH,
+  TRANSLITERATION_DEVNAGRI,
+  TRANSLITERATION_URDU,
+  TRANSLATION_ENGLISH,
+  TRANSLATION_PUNJABI,
+  TRANSLATION_SPANSISH,
+} from './lib/mool-mantar'
+
+loadStorage()
+loadCss()
+
+const App = () => {
+  const settingsState = useState( {} )
+
+  return (
+    <SettingsContext.Provider value={settingsState}>
+
```

Comment:
double newline? throw in the desktop eslint to get rid of most of these problems